### PR TITLE
Documentation typo

### PR DESCRIPTION
**Pull request by stphnpt**
`Sonata\PageBundle\Block\BlockServiceInterface` does not exist

### CODE REVIEW

File: Resources/doc/reference/introduction.rst [hunk 1]
```diff
@@ -73,6 +73,9 @@ By default the ``SonataPageBundle`` is shipped with core block services:
 
  - sonata.page.block.container      : Block container
  - sonata.page.block.children_pages : Render a navigation panel
+ - sonata.page.block.breadcrumb     : Render a breadcrumb
```
Comment (Soullivaneuh) [inline Resources/doc/reference/introduction.rst]: This is not related. Please do that on another bundle or at least, another commit.

Comment (stphnpt) [inline Resources/doc/reference/introduction.rst]: OK. I think the doc would be richer if the 5 block services existing in the page bundle were listed here (as proposed), no ?

Comment (Soullivaneuh) [inline Resources/doc/reference/introduction.rst]: I didn't say your addition is useless. I said your addition is **not related** to your PR/commit.

cc @greg0ire

Comment (greg0ire) [inline Resources/doc/reference/introduction.rst]: @stphnpt : use `git add --patch` to commit different parts of a file separately. It's also a great tool to review your own code before committing it.

**Comment by OskarStark** [comment]
Thank you @stphnpt 👍
